golang/update_protos.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
```

Comment:
@bdferris-v2 we actually have to run the script from the project root because of the docker build context. `docker build` cannot pull in files that are not siblings (or children of siblings) of the executing context (where one executing the `docker` command). Because the `gtfs-realtime.proto` file is located at the project root, we must execute `docker build` from there

Reply:
Ah, I missed that.  Then I'm going to double down on this advice :)  I think it's likely that someone will attempt to run the `update_protos.sh` script from the `golang` directory and then be a bit confused why the docker build fails, complaining about missing directories.

So I'd propose something like:

GOLANG_DIR=$(dirname "$0")
ROOT_DIR=$(dirname $GOLANG_DIR)

and pass $ROOT_DIR to the `docker build` command instead of `.` to be explicit about the build directory.  Thoughts?

Reply:
@bdferris-v2 Alrighty, I pushed up another update that I _think_ should do the trick. We needed to use ``$(cd `dirname $0` && pwd)`` rather than simply using `dirname` to account for the case where we execute from within the `golang` dir itself, since the initial `dirname` would result in `.`, and `dirname "."` just results in `.` again rather than `..`.

I believe that grabbing the absolute path should allows us to handle execution from anywhere!